### PR TITLE
Travis CI: fix badge, create missing directories for tests

### PR DESCRIPTION
**Pull request by Roman3349**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ install:
   - travis_retry composer install --no-progress --prefer-dist
 script:
   # Tests
+  - mkdir tests/acceptance tests/functional || true
```
Comment (MartkCz) [inline .travis.yml]: imho It's better to create directories and add .gitkeep

Comment (Roman3349) [reply .travis.yml]: @MartkCz Fixed

**Comment by MartkCz** [comment]
@f3l1x

**Comment by f3l1x** [comment]
👍

**Pull request merged**